src/vivarium_public_health/population/base_population.py

```diff
@@ -207,22 +207,43 @@ def _load_population_structure(self, builder: Builder) -> pd.DataFrame:
 
 
 class ScaledPopulation(BasePopulation):
-    """Component for scaling the population structure based on a scaling factor."""
+    """Component for scaling the population structure based on a scaling factor.
 
-    def __init__(self, scaling_factor: str | float | pd.DataFrame):
+    This component is to be used in place of the `BasePopulation` component when
```

Comment:
This is a loooooong sentence with a lot of the word "population". What about simply:

`This component is to be used in place of `BasePopulation` when all simulants are a subset of the total population and need to be scaled up.`